db/migrations/campaign/V4.0.0.4__table_campaigns.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS campaign.beneficiaries (
+    id uuid NOT NULL DEFAULT (gen_random_uuid()),
+    name character varying(100) NOT NULL,
+    type integer NOT NULL,
+    iso2country_code integer NOT NULL,
```

Comment:
We can store country code as is (string representation). This number is not very self-explanatory.